react/legacy/components/ProductSummaryPrice.js

```diff
@@ -25,6 +38,37 @@ const ProductSummaryPrice = ({
     )
   }
 
+  const sortPrices = (priceA, priceB) => priceA - priceB
+  const lowestPrice = compose(head, sort(sortPrices))
+  const highestPrice = compose(last, sort(sortPrices))
+
+  const getListPrices = prices => {
+    const lowPrice = lowestPrice(prices)
+    const highPrice = highestPrice(prices)
+    return [
+      lowPrice,
+      highPrice,
+    ]
+  }
+
+  const isAvailableProduct = price => price !== 0
+
+  const getPriceRange = () => {
+    const items = prop('items', product)
+    if (items) {
+      const sellers = flatten(pluck('sellers', items))
+      const prices = map(path(['commertialOffer', 'Price']), sellers)
+      const availableProductsPrices = filter(isAvailableProduct, prices)
+      
+      return getListPrices(availableProductsPrices)
+    }
+
+    return []
+  }
+
+  const priceRange = getPriceRange()
+  const showPriceRange = equals(uniq(priceRange), priceRange)
```

Comment:
change this to:
`const showPriceRange = priceRange.length === 2 && priceRange[0] !== priceRange[1]`

Reply:
it's better to use `destructuring`, right?
so, we have 
```js
const [lowPrice, highPrice] = priceRange
const showPriceRange = lowPrice !== highPrice
```
It's always garanteed that we have two values, even if they are equals, this is because we make this check

Reply:
it's better use `destructuring`, right?
```js
const [lowPrice, highPrice] = priceRange
const showPriceRange = lowPrice !== highPrice
```
Even if `priceRange` is empty, the lowPrice and highPrice should be undefined, then our check would work.